File: 136-single-number/single-number.cpp
class Solution {
public:
    int singleNumber(vector<int>& nums) {
        int n = nums.size();
        int count = 0;
        for(int i = 0; i<n;i++)
        {
            count = count ^ nums[i];
        }
        return count;
    }
};

File: 136-single-number/single-number.py
class Solution:
    def singleNumber(self, nums: List[int]) -> int:
        d = {}
        for num in nums:
            if num not in d:
                d[num] = 0
            d[num] += 1
        for num in d:
            if d[num] == 1:
                return num